### PR TITLE
Clean up Math Symbols in Math Editor

### PR DESCRIPTION
**Pull request by hedgerwang**
Per request from users, we need to clean up some math symbols and add more symbols.
This diff updates the symbols used in math editor.

| before | after |
|---|---|
| image image | image image | 
|---|---|

### CODE REVIEW

File: src/ui/mathquill-editor/MathQuillEditorSymbols.js [hunk 1]
```diff
@@ -7,229 +7,467 @@ export type MathQuillEditorSymbol = {
...
   label: '\\bigtriangleup',
   latex: '\\bigtriangleup',
   description: 'Triangle',
   cmd: 'write',
 };
-const X = {label: 'x', latex: 'x', description: 'x', cmd: 'write'};
-const Y = {label: 'y', latex: 'y', description: 'y', cmd: 'write'};
-
+export const UNDERLINE = {
+  label: '\\underline\u3000',
```
Comment (velveret) [inline src/ui/mathquill-editor/MathQuillEditorSymbols.js]: `label` intentionally differs from the `latex` in this one?

File: src/ui/mathquill-editor/MathQuillEditorSymbols.js [hunk 2]
```diff
@@ -7,229 +7,467 @@ export type MathQuillEditorSymbol = {
...
+  cmd: 'write',
+};
+export const SMALLE = {
+  label: '\u212F',
+  latex: '\u212F',
+  description: 'Script Small E',
+  cmd: 'write',
+};
+export const SQR = {
+  label: 'x^{2}',
```
Comment (velveret) [inline src/ui/mathquill-editor/MathQuillEditorSymbols.js]: `label` intentionally different from `latex`?

File: src/ui/mathquill-editor/MathQuillEditorSymbols.js [hunk 3]
```diff
@@ -7,229 +7,467 @@ export type MathQuillEditorSymbol = {
...
+export const PM = {
   label: '\\pm',
   latex: '\\pm',
   description: 'Plus-Minus',
   cmd: 'cmd',
 };
-const SIN = {label: '\\sin', latex: '\\sin', description: 'Sin', cmd: 'write'};
-const SQRT2 = {
+export const POWER = {
+  label: 'x^{y}',
```
Comment (velveret) [inline src/ui/mathquill-editor/MathQuillEditorSymbols.js]: label intentionally different from latex?

File: src/ui/mathquill-editor/MathQuillEditorSymbols.js [hunk 4]
```diff
@@ -7,229 +7,467 @@ export type MathQuillEditorSymbol = {
...
+export const NEQ = {
+  label: '\\neq',
+  latex: '\\neq',
+  description: 'Not Equal',
   cmd: 'write',
 };
-const PARENS = {
+
+export const OVERLINE = {
+  label: '\\overline\u3000',
```
Comment (velveret) [inline src/ui/mathquill-editor/MathQuillEditorSymbols.js]: label intentionally different from latex?

File: src/ui/mathquill-editor/MathQuillEditorSymbols.js (hunk 1 above)
Comment (hedgerwang) [reply src/ui/mathquill-editor/MathQuillEditorSymbols.js]: Yes.

`\u3000'` shows the wide space character and the label will look like "___".
`\\underline` is just the LateX style and "\u3000" is the text to be styled (with underline).

File: src/ui/mathquill-editor/MathQuillEditorSymbols.js (hunk 3 above)
Comment (hedgerwang) [reply src/ui/mathquill-editor/MathQuillEditorSymbols.js]: Yes.

**Pull request merged**